### ISSUE TITLE
regression on main: ModuleNotFoundError: No module named 'jaxlib.mlir.extras'

### ISSUE THREAD

**Issue by brettkoonce** [google/jax#19112]
hitting this with b413ee2549f70dd304bff3e9b84fccdc4b4c9049 (current head)

  pip install /home/skoonce/jax-2023/jax/dist/jaxlib-0.4.24.dev20231224-cp310-cp310-manylinux2014_x86_64.whl --force-reinstall                                  
...
Successfully installed jaxlib-0.4.24.dev20231224 ml-dtypes-0.3.1 numpy-1.26.2 sc
ipy-1.11.4

install jax (eg pip install .)
Successfully installed jax-0.4.24.dev20231223+b413ee254

```
python3
Python 3.10.13 (main, Sep 11 2023, 13:44:35) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import jax
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/skoonce/jax-2023/jax/jax/__init__.py", line 85, in <module>
    from jax._src.api import effects_barrier as effects_barrier
  File "/home/skoonce/jax-2023/jax/jax/_src/api.py", line 39, in <module>
    from jax._src import stages
  File "/home/skoonce/jax-2023/jax/jax/_src/stages.py", line 46, in <module>
    from jax._src.interpreters import mlir
  File "/home/skoonce/jax-2023/jax/jax/_src/interpreters/mlir.py", line 52, in <module>
    from jax._src.lib.mlir import dialects
  File "/home/skoonce/jax-2023/jax/jax/_src/lib/mlir/dialects/__init__.py", line 17, in <module>
    import jaxlib.mlir.dialects.builtin as builtin
  File "/home/skoonce/anaconda3/envs/jax-2023/lib/python3.10/site-packages/jaxlib/mlir/dialects/builtin.py", line 9, in <module>
    from ..extras.meta import region_op
ModuleNotFoundError: No module named 'jaxlib.mlir.extras'
```

a few commits back works here:
b9aa5897094fb8b6cd35b90e600758a905420ca2

python3 build/build.py --enable_cuda

install built jaxlib + jax

python3
Python 3.10.13 (main, Sep 11 2023, 13:44:35) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import jax
>>> jax.devices()
[cuda(id=0), cuda(id=1), cuda(id=2), cuda(id=3)]
>>>

**Comment by brett-koonce** [comment]
fixed by c83fd971a0e6ec6726f255fd93937a0522f5f916!

**Comment by brett-koonce** [comment]
this can be closed, don't have permissions for it!
